apps/notifications/src/app/note-shortcuts/style.scss

```diff
@@ -0,0 +1,15 @@
+.wpnc__keyboard-shortcuts-popover {
```

Comment:
I'm not sure if we still need to follow v1's class names; I'm just following existing convention in the v2 notifications 🙈